widgyts/dataset_viewer.py

```diff
@@ -0,0 +1,252 @@
+import numpy as np
+import json
+import pythreejs
```

Comment:
so, pythreejs is a hard dependency for us now. Do we want to use it for other widgyts that we build out too?